src/dlgConnectionProfiles.h

```diff
@@ -31,21 +31,17 @@ class dlgConnectionProfiles : public QDialog , public Ui::profile_dialog
     Q_OBJECT
 
 public:
-
     dlgConnectionProfiles(QWidget * parent = 0);
     void fillout_form();
-    void copy_profile( QString );
     void writeProfileData( QString, QString, QString );
     QString readProfileData( QString, QString );
     QStringList readProfileHistory( QString, QString );
     void accept() override;
 
 signals:
-
     void signal_establish_connection( QString profile_name, int historyVersion );
```

Comment:
:thinking: I believe `int historyVersion` is unused throughout the whole application.